util/testutil.cc

```diff
@@ -2,9 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
-#include "util/testutil.h"
```

Comment:
This is incorrect.

Every .cc file starts out by including its corresponding header. This lets the compiler that

1. Definitions in the .cc match the declarations in the .h file
2. The .h file compiles without any external #includes.